install-and-configure/install/multi-cluster/thanos-setup/thanos-upgrade.md

```diff
@@ -23,7 +25,7 @@ config:
   access_key: ${your_access_key}
   insecure: false
   signature_version2: false
-  #encrypt_sse: false <--- THIS LINE NEEDS TO BE DELETED
+  #encrypt_sse: false <= Remove this line!
```

Comment:
Use YAML code syntax in the fencing. Also, comments are valid in YAML so what's shown here (commented out) has the same effect as its absence.